akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala

```diff
@@ -104,6 +104,9 @@ class PhiAccrualFailureDetector(
 
   private val acceptableHeartbeatPauseMillis = acceptableHeartbeatPause.toMillis
 
+  // FIXME: fix this binary compatibility hack in 2.6
+  private[akka] var address: String = "N/A"
```

Comment:
This isn't thread, doesn't it need to be?

Reply:
It's set once when right after the instance is created so that should be fine. If you think about visibility it should become visible soon by other memory barriers.

Reply:
👍